gci/models.py

```diff
@@ -1 +1,109 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Students(models.Model):
```

Comment:
It is advisable and generally followed that all models are named in **singular** nouns. Please rename all other models too.